.circleci/config.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     steps:
       - checkout
       - run: docker pull mesg/core:latest
-      - run: docker build -t mesg/core:$CIRCLE_TAG -t mesg/core:latest .
+      - run: docker build -t mesg/core:$CIRCLE_TAG -t mesg/core:latest --build-arg version=`echo $CIRCLE_SHA1 | cut -c1-7` .
```

Comment:
In this step (`publish_docker_prod`) we want to use the version of the tag (`$CIRCLE_TAG`)

Reply:
But you forget to add the version based on the sha1 in the step `publish_docker_dev`:
https://github.com/mesg-foundation/core/blob/a880f34c273793004dab254061b888cdeb41ff42/.circleci/config.yml#L45-L52

You have to add the version based on the sha1 with a nice prefix like:
https://github.com/mesg-foundation/core/blob/a880f34c273793004dab254061b888cdeb41ff42/.circleci/config.yml#L68

Reply:
I didn't put version in docker_dev because I didn't know what version should we use there,

About docker prod, this is version for core binary and I took it from cli build. Do we want diffrent version for core & cli?

Reply:
Look at the step to build the cli.
For dev build, we use as version: `dev build `echo $CIRCLE_SHA1 | cut -c1-7``
For prod build, we use: `$CIRCLE_TAG`

Reply:
ok